### DISCUSSION TITLE
Basics // Table alignment

### DISCUSSION THREAD

**Discussion by LeHarry** [frappe/print_designer#173] [Q&A] [answered]
Q: How to align tables in Printdesigner properly so that items are left aligned and and numbers are right aligned?
Q: How to add/delete columns?

![grafik](https://github.com/frappe/print_designer/assets/160015230/939d58d0-26fc-484b-9f33-8ef011d42afb)

**Comment by maharshivpatel** [comment, marked as answer]
@LeHarry for column level styling  https://github.com/frappe/print_designer/pull/158. 

open context menu like this to add or delete columns : https://youtu.be/XpmrLzbianE?si=U64BC8N5vM5eCmUP&t=570
